tests/artemis-test-support/src/main/java/org/apache/activemq/transport/amqp/client/AmqpProtocolTracer.java

```diff
@@ -37,7 +39,7 @@
  */
 public class AmqpProtocolTracer implements ProtocolTracer, FrameBodyHandler<AmqpFrameValidator> {
 
-   private static final Logger TRACE_FRAMES = LoggerFactory.getLogger(AmqpProtocolTracer.class.getPackage().getName() + ".FRAMES");
+   private static final Logger logger = LoggerFactory.getLogger(MethodHandles.lookup().lookupClass());
```

Comment:
Noting it changes the logger name. Maybe not an issue as its only the test client.

Reply:
I think we should keep the change here.. it's better this way.. as you said it's on the test client.